ffi/capi/src/properties_enums.rs

```diff
@@ -72,10 +75,57 @@ pub mod ffi {
     }
 
     impl BidiClass {
-        pub fn to_integer(self) -> u8 {
+        #[diplomat::rust_link(icu::properties::props::EnumeratedProperty::for_char, FnInTrait)]
+        #[cfg(feature = "compiled_data")]
+        pub fn for_char(ch: DiplomatChar) -> Self {
+            icu_properties::CodePointMapData::<props::BidiClass>::new()
+                .get32(ch)
+                .into()
+        }
+
+        #[diplomat::rust_link(icu::properties::PropertyNamesLongBorrowed::get, FnInStruct)]
+        #[diplomat::rust_link(icu::properties::PropertyNamesLong, Struct, hidden)]
+        #[diplomat::rust_link(icu::properties::PropertyNamesLongBorrowed, Struct, hidden)]
+        #[diplomat::rust_link(icu::properties::PropertyNamesLong::new, FnInStruct, hidden)]
+        #[diplomat::rust_link(icu::properties::PropertyNamesLongBorrowed::new, FnInStruct, hidden)]
+        #[diplomat::rust_link(
+            icu::properties::props::NamedEnumeratedProperty::long_name,
+            FnInTrait,
+            hidden
+        )]
+        #[cfg(feature = "compiled_data")]
+        /// Get the "long" name of this property value (returns empty if property value is unknown)
+        pub fn long_name(self) -> &'static str {
+            icu_properties::PropertyNamesLongBorrowed::<props::BidiClass>::new()
+                .get(self.into())
+                .unwrap_or("")
+        }
+
+        #[diplomat::rust_link(icu::properties::PropertyNamesShortBorrowed::get, FnInStruct)]
+        #[diplomat::rust_link(icu::properties::PropertyNamesShort, Struct, hidden)]
+        #[diplomat::rust_link(icu::properties::PropertyNamesShortBorrowed, Struct, hidden)]
+        #[diplomat::rust_link(icu::properties::PropertyNamesShort::new, FnInStruct, hidden)]
+        #[diplomat::rust_link(icu::properties::PropertyNamesShortBorrowed::new, FnInStruct, hidden)]
+        #[diplomat::rust_link(
+            icu::properties::props::NamedEnumeratedProperty::short_name,
+            FnInTrait,
+            hidden
+        )]
+        #[cfg(feature = "compiled_data")]
+        /// Get the "short" name of this property value (returns empty if property value is unknown)
+        pub fn short_name(self) -> &'static str {
+            icu_properties::PropertyNamesShortBorrowed::<props::BidiClass>::new()
+                .get(self.into())
+                .unwrap_or("")
+        }
+        #[diplomat::rust_link(icu::properties::props::BidiClass::to_icu4c_value, FnInStruct)]
+        /// Convert to an integer value usable with ICU4C and CodePointMapData
+        pub fn to_integer_value(self) -> u8 {
```

Comment:
should we use the same names?

```suggestion
        pub fn to_icu4c_value(self) -> u8 {
```